### PR TITLE
Title and description support

### PR DESCRIPTION
**Pull request by sterlp**
- Added support for title and description 
- fixed smaller CLI issues
- Doc update how to use locally

### CODE REVIEW

File: lib/cli.js [hunk 1]
```diff
@@ -58,7 +57,9 @@ Options:
  -p, --pretty            pretty-print JSON result
  --scheme                scheme (default: http)
  -t, --target            target file (default: source file basename + .openapi3.json)
- --skipBatchPath         skips the generation of the $batch path, (default: false)`);
+ --skipBatchPath         skips the generation of the $batch path, (default: false)
+ --title                 custom title which should be used instead of the default one
+ --description           custom description which should be used instead of the default one`);
```
Comment (ralfhandl) [inline lib/cli.js:62]:
```suggestion
 --title                 default title if none is annotated
 --description           default description if none is annotated`);
```

File: lib/README.md [hunk 2]
```diff
@@ -16,6 +16,24 @@ To install globally type
 npm install -g odata-openapi
 ```
 
+## Local installation of the CLI
```
Comment (ralfhandl) [inline lib/README.md:19]: npm link is not quite "local installation", see https://docs.npmjs.com/cli/v9/commands/npm-link

**Review by ralfhandl — COMMENTED** [review]
See comments

File: lib/README.md (hunk 2 above)
Comment (sterlp) [reply lib/README.md]: adjusted

**Review by HeikoTheissen — APPROVED** [review]

**Comment by ralfhandl** [comment]
Published at https://www.npmjs.com/package/odata-openapi

**Pull request merged**